src/app/client/src/app/modules/discussion/discussion.module.ts

```diff
@@ -28,9 +28,9 @@ export class DiscussionModule {
     if (!CsModule.instance.isInitialised) {
       this.csLibInitializerService.initializeCs();
     }
-    this.discussionEvents.telemetryEvent.subscribe((event) => {
-      this.discussionTelemetryService.logTelemetryEvent(event);
-    });
+    // this.discussionEvents.telemetryEvent.subscribe((event) => {
+    //   this.discussionTelemetryService.logTelemetryEvent(event);
+    // });
```

Comment:
@princegupta1131 
remove the commented code if not required

Reply:
Removed the commeted line